# HG changeset patch
# User alta88@fixall.com
# Date 1763581729 -3600
# Parent  627223a5272abb8db36d6843c274fa7486e80a5b
QFB fixes for TB changes in 128:
1. Tab stop on clear button

diff --git a/mail/base/content/widgets/search-bar.inc.xhtml b/mail/base/content/widgets/search-bar.inc.xhtml
--- a/mail/base/content/widgets/search-bar.inc.xhtml
+++ b/mail/base/content/widgets/search-bar.inc.xhtml
@@ -10,10 +10,9 @@
     <button id="clear-button"
             type="reset"
             class="button button-flat icon-button"
             data-l10n-id="search-bar-clear-button"
-            hidden="hidden"
-            tabindex="-1">
+            hidden="hidden">
       <slot name="clear-button"></slot>
     </button>
     <button id="search-button"
             class="button button-flat icon-button"
diff --git a/mail/components/storybook/stories/search-bar.stories.mjs b/mail/components/storybook/stories/search-bar.stories.mjs
--- a/mail/components/storybook/stories/search-bar.stories.mjs
+++ b/mail/components/storybook/stories/search-bar.stories.mjs
@@ -32,9 +32,8 @@ const Template = ({ label, disabled }) =
         type="reset"
         data-l10n-id="search-bar-clear-button"
         class="button button-flat icon-button"
         hidden="hidden"
-        tabindex="-1"
       >
         <slot name="clear-button"></slot>
       </button>
       <button id="search-button" class="button button-flat icon-button">
